### PR TITLE
Add Rack Statuses doc

### PR DESCRIPTION
**Pull request by ianmalott**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

File: _reference/rack-statuses.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Rack Statuses"
+---
+
+As you interact with your Rack, you will notice a variety of statuses displayed in the Console web interface and in the output of CLI commands like `convox rack`. Here's what they mean.
+
+## converging
+
+Your Rack is running, but it is rearranging itself. For example, you might see the _converging_ status during a routine instance replacement, or when a service is rescheduled in the cluster.
```
Comment (mattmanning) [inline _reference/rack-statuses.md]: Please mention app deployments specifically here. I think that could replace "when a service is rescheduled in the cluster" which might be a little too much AWS jargon.

File: _reference/rack-statuses.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+If a Rack update fails to complete, or is canceled, the Rack will display a status of _rollback_ while it reverts to the last known good state.
+
+## running
+
+Most of the time, your Rack should display the _running_ status and be fully operational.
+
+## unreachable
+
+When Console or your CLI are unable to communicate with a Rack, that Rack will have a status of _unreachable_. Authentication issues, like bad passwords, are often the problem, but an uninstalled Rack will also be listed as _unreachable_ in Console until it is manually removed via its settings page.
```
Comment (mattmanning) [inline _reference/rack-statuses.md]: Do we have a doc that explains how to fix a bad password? If so can you link it here?

File: _reference/rack-statuses.md (hunk 1 above)
Comment (ianmalott) [inline _reference/rack-statuses.md]: 👍

File: _reference/rack-statuses.md (hunk 2 above)
Comment (ianmalott) [inline _reference/rack-statuses.md]: 👍

**Comment by ianmalott** [comment]
@mattmanning ready for another look

**Comment by mattmanning** [comment]
@mwarkentin Great feedback, thanks. I added another sentence to the _converging_ section to mention how to check for apps that are still updating.

We are going to add another doc specifically about app statuses and are working on getting more of the ECS info into convox logs. I think those efforts will provide more of the information you're looking for.

**Comment by mwarkentin** [comment]
:+1:

**Pull request merged**